CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+## v1.0.0 (02/12/2019)
+
+**Common**
+
+* Project structure was changed, public API aligned to the HERE OLP SDK for C++.
+* Unit tests was improved
+
+**olp-sdk-dataservice-read**
+
+* Added new class `OlpClientSetting` with `KeyValueCache` instance to be used for context
+* Added new class `VersionedLayerClient` that is used to access versioned layers in OLP. Class implements `GetData`, `GetPartitions` methods.
+* Added new class `VolatileLayerClient` that is used to access volatile layers in OLP. Class implements `GetPartitions`, `GetData` methods.
+* Added new class `ArtifactClient` that is used to access artifact service in OLP. Class implements `getSchemaDetails`, `getSchema` methods.
+* Added new class `CatalogClient` that is used to access catalog metada in OLP. Class implements `getCatalog`, `getLatestVersion`, `getVersions` methods.
+* Added new class `ConfigClient` that is used to access config service in OLP. Class implements `getCatalogs` method.
+
+
+**Breaking Change**
+
+* CatalogClient constructor changed. `OlpClientSettings` must be passed by value, `KeyValueCache` is now part of `OlpClientSettings`.
+* HypeDataProvider removed.
+* DatastoreClient removed.
+* Methods getTile and getAgregatedTile removed.
```

Comment:
From were removed and why? What is the alternative method to use?

Reply:
updated